prusti-viper/src/encoder/mir/procedures/encoder/mod.rs

```diff
@@ -1488,11 +1488,9 @@ impl<'p, 'v: 'p, 'tcx: 'v> ProcedureEncoder<'p, 'v, 'tcx> {
         // The called method might be a trait method.
         // We try to resolve it to the concrete implementation
         // and type substitutions.
+        let query = self.encoder.env().query;
```

Comment:
Why this alias? It doesn't seem to be used later on.

Reply:
Without it, the long line spilled over into a silly-looking chained function call spanning like 5 lines, lol. I considered updating the later usage, but that felt worse. Should I just accept the other formatting rather than polluting this long function's namespace?